tests/testthat/test-FilteredData.R

```diff
@@ -319,6 +326,41 @@ testthat::test_that("get_filter_state returns `teal_slices` with features identi
 
   fs_out <- unname(shiny::isolate(datasets$get_filter_state()))
 
+  expect_identical_slices(fs, fs_out)
+})
+
+# format ----
+testthat::test_that("format returns properly formatted string representing `teal_slices` with class header", {
+  datasets <- FilteredData$new(
+    list(
+      iris = list(dataset = iris),
+      mtcars = list(dataset = mtcars)
+    )
+  )
+
+  fs <- teal_slices(
+    teal_slice(
+      dataname = "iris", varname = "Sepal.Length",
+      choices = c(4.3, 7.9), selected = c(5.1, 6.4),
+      keep_na = FALSE, keep_inf = FALSE
+    ),
+    teal_slice(
+      dataname = "iris", varname = "Species",
+      choices = c("setosa", "versicolor", "virginica"), multiple = TRUE, selected = c("setosa", "versicolor"),
+      keep_na = FALSE
+    ),
+    teal_slice(
+      dataname = "mtcars", varname = "cyl",
+      choices = c("4", "6", "8"), multiple = TRUE, selected = c("4", "6"),
+      keep_na = FALSE, keep_inf = FALSE
+    ),
+    count_type = "none",
+    include_varnames = list(mtcars = "cyl"),
+    exclude_varnames = list(iris = c("Petal.Length", "Petal.Width"))
+  )
+
+  datasets$set_filter_state(state = fs)
+
   testthat::expect_identical(
     shiny::isolate(datasets$format()),
     paste0("FilteredData:\n", format(fs_out))
```

Comment:
```suggestion
    paste0("FilteredData:\n", format(fs))
```
fs_out does not exist in this test.

Reply:
same for below test as well.